CHOLMOD/GPU/t_cholmod_gpu.c

```diff
@@ -188,6 +188,9 @@ int TEMPLATE2 (CHOLMOD (gpu_init))
 
 }
 
+#if !defined(NCOMPAR_FN_T)
+typedef int (*__compar_fn_t) (const void*, const void*);
+#enfif
```

Comment:
``` suggestion
#endif
```

Reply:
Fixed

Reply:
The suggestion has been applied on another computer using another account. Here is a typo.

Reply:
Where did you fix it? It isn't visible in the PR (yet).